### DISCUSSION TITLE
How is everyone doing?

### DISCUSSION THREAD

**Discussion by kanjurer** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#176] [General]
How is the job hunt going for Summer 2024?

**Comment by ShevinuM** [comment]
Nothing for me. No interviews or offers (except some digital interviews). Wbu?

**Reply by kanjurer** [reply]
Haha same old same old! There aren't many jobs this time around to begin with.

**Reply by kanjurer** [reply]
Although, I did get a "return offer" from my current workplace for full-time xD so that is fun. I am planning on doing some research at uni this summer + work on some _actual_ cool projects (which I never got to do cause I was always busy with school/internships).

**Reply by ShevinuM** [reply]
That's huge dude. Congrats
